### DISCUSSION TITLE
collections of datasets based on previously available data

### DISCUSSION THREAD

**Discussion by acarolinafilipe** [dandi/helpdesk#146] [Q&A] [answered]
I'd like to know if it's possible to publish datasets on Dandi that are collections of other datasets (carefully and thoughtfully designed). If so, how could I credit the original authors? What is the policy for sharing this type of dataset?

**Comment by yarikoptic** [comment]
First and foremost it depends on the licenses/terms and conditions of use of those original datasets -- do they allow for derived works and re-sharing?  What do their license(s) mandate in terms of acknowledgement?  Are their licenses compatible (ie. not having conflicting terms which would prohibit combination of components from multiple under some "common denominator" license)?

**Reply by acarolinafilipe** [reply]
Their licenses are CC BY 4.0 and CC0 1.0

**Reply by satra** [reply]
just to clarify further, do you mean collections of other dandisets? or other datasets elsewhere?

**Reply by acarolinafilipe** [reply]
Both, it's very similar to Neural Latent Benchmark. Most of the original datasets are the same, but in my case I processed the datasets in their entirety and not just one session.

**Reply by satra** [reply, marked as answer]
thanks for the clarification. then there are no restrictions. you will likely want to use the dandiset metadata (related resources) to indicate the original sources of these datasets. in addition, you may want to add a README.md file as well with details.

**Reply by yarikoptic** [reply]
Minor: I think you better select CC BY 4.0 for your dataset as it is the most restrictive among the ones you are combining. State the original copyrights (if stated, if not -- at least authors) whenever you state your copyright/license.
